### ISSUE TITLE
18.04.2 image for Raspberry Pi 4 isn't compatible

### ISSUE THREAD

**Issue by swinterbotix** [ubuntu-mate/ubuntu-mate.org#86]
Hi,

I've tried to use the 64-bit Ubuntu 18.04.2 image for the Raspberry Pi 4 at https://ubuntu-mate.org/download/arm64/bionic/, but it does not seem to work. I've confirmed it works with the Pi 3, but it does not seem to work for the Pi 4. I also noticed that the file name associated with the download is ubuntu-mate-18.04.2-beta1-desktop-arm64+raspi3-ext4.img.xz. The fact that 'raspi3' is in the file name leads me to believe that there should be a different file name for the raspberry 4 called 'raspi4'. Either that, or maybe just a generic 'raspi' or something? Was this link just not updated?

**Comment by lah7** [comment]
Good point, thanks for letting us know.

It's actually a bit of an oversight, due to the way the downloads on the website are populated. The 18.04.2 image is not compatible with the Raspberry Pi 4 out of the box. Prior to adding 20.04, all the references were Pi 2 and 3.

Alongside #77, we will need to overhaul the downloads page to address how we present them to resolve this from the website's point of view.

---

The alternate solution would be if the project released an updated 18.04 image, adding compatibility for Pi 4, but you'll have to ask the [community](https://ubuntu-mate.community) or [Wimpy's World on Discord](https://wimpysworld.io/discord).

**Comment by thillRobot** [comment]
I have noticed that since you overhauled the downloads page the **ubuntu-mate-18.04.2-beta1-desktop-arm64+raspi3-ext4.img.xz** is no longer available. Is this not supported until April 2021? I know things are moving on, but I am still working on some 18.04 systems.

**Comment by lah7** [comment]
The 18.04 downloads for the Raspberry Pi were actually dropped by the leader recently (c141316fc2e52b0bd656c1d6736ef55e70b47aac), but you can still obtain them at https://releases.ubuntu-mate.org/archived/bionic/ under `armhf` or `arm64`.

Since the underlying base is Ubuntu, these systems will still be supported for the life of Ubuntu 18.04 "Bionic". Type this command for coverage:

    ubuntu-support-status

**Comment by thillRobot** [comment]
Ah. I looked under releases, but I overlooked the `armhf` directory at the top of the page. I will look more carefully in the future. Thanks for the information.

**Comment by lah7** [comment]
It's not a great directory listing, but it is something I'd like to improve in future. (Noting this in #97 :memo:)

**Comment by madhens** [comment]
Summarizing this info from the #web channel on Discord, to include in the updated section:
armhf - suitable for low RAM devices Pi2 and Pi3. Will possibly be deprecated in the future
arm64 - sensible on devices with 4GB RAM and up (even 2GB and up for Ubuntu MATE)
recommend Pi 4 with at least 2GB RAM using arm64
The new hardware enablement will favour arm64, memory pressure is relived and CPU performance is significantly better than even the Pi3 B+

**Comment by lah7** [comment]
New information made its way in 4154a8fb28fcc4ddcb4223638f5582401ed7d9c0 (`rpi-update` branch). I added a table [(then turned into YAML)](https://github.com/ubuntu-mate/ubuntu-mate.org/commit/f92219eb5a442a7206d3c70857bc3a2538264d25) to clearly show which images are compatible.

I just realised the new table only shows armhf vs arm64. We might need another to show Pi models against releases (e.g. 18.04 does not work on Raspberry Pi 4) and any caveats for new/old images.
